test/regress/cli/regress0/options/safe-options-fatal.smt2

```diff
@@ -0,0 +1,9 @@
+; DISABLE-TESTER: dump
+; REQUIRES: no-competition
+; EXPECT: (error "Fatal error in option parsing: expert option global-negate cannot be set when safeOptions is true.")
```

Comment:
The error message is slightly inconsistent: global-negate vs safeOptions. Maybe use safe-options?